### PR TITLE
chem.json

### PR DESCRIPTION
**Pull request by allaway**
Added a file with cheminformatics relevant annotations. Definitely more to do, but this is a minimal set that covers my use cases.

### CODE REVIEW

**Comment by allaway** [comment]
sorry @kdaily for inadvertently leaving you out of this one :)
@sgosline had proposed merging this with compoundScreen.json.

File: synapseAnnotations/data/chem.json [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        "description": "The popular 166 bit MACCS keys described by MDL.", 
+        "source": "https://www.rdocumentation.org/packages/rcdk/versions/3.4.5/topics/get.fingerprint"
+      },
+      {
+        "value": "pubchem", 
+        "description": "881 bit fingerprints defined by PubChem.", 
+        "source": "https://www.rdocumentation.org/packages/rcdk/versions/3.4.5/topics/get.fingerprint"
+      },
+      {
+        "value": "klekota and roth",
```
Comment (kdaily) [inline synapseAnnotations/data/chem.json]: capitalize

File: synapseAnnotations/data/chem.json [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        "description": "Specification for unambiguously describing the structure of a chemical compound using a short ASCII string.", 
+        "source": "http://purl.obolibrary.org/obo/MS_1000868"
+      },
+      {
+        "value": "InChI", 
+        "description": "A textual identifier for chemical substances designed to provide a standard and human-readable way to encode molecular information that also facilitates searches in printed and electronic data sources.", 
+        "source": "http://purl.obolibrary.org/obo/NCIT_C54683"
+      },
+      {
+        "value": "sdf",
```
Comment (kdaily) [inline synapseAnnotations/data/chem.json]: capitalize per EDAM: `SDF`

File: synapseAnnotations/data/chem.json (hunk 1 above)
Comment (allaway) [reply synapseAnnotations/data/chem.json]: @kdaily these are not capitalized in the source documentation. Did change "klekota and roth" to "kr" to mirror source docs. What should I do?

**Comment by allaway** [comment]
woo!

**Pull request merged**